model/business-process-new/base.js

```diff
@@ -79,7 +79,88 @@ module.exports = memoize(function (db/*, options*/) {
 			if (this.manager) result.push(_observe(this.manager._email));
 
 			return result;
-		} }
+		} },
+		toWebServiceJSON: {
+			value: function (ignore) {
+				var dataFields = {}, result, db;
+				db = this.database;
+				result = {
+					id: this.__id__,
+					service: {
+						code: this.constructor.__id__
+					},
+					submittedTimestamp:
+						this._isSubmitted.lastModified ?
+								Math.floor(this._isSubmitted.lastModified / 1000) : null,
+					processingSteps: {},
+					request: {
+						registrations: [],
+						documentUploads: [],
+						costs: [],
+						payments: [],
```

Comment:
I don't think we have to use our internal name here, payments seems a better (and suggested in doc) name for this functionality.